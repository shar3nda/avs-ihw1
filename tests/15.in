12
1 5 9 4 7 -2 10 1 4 9 -2 2