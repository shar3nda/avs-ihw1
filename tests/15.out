-2 10 1 4 9 -2 2